### PR TITLE
chore(android): google pay related cleanup and fixes

### PR DESCRIPTION
**Pull request by felipecsl**
## Summary

* Replace `mavenLocal` with a project directory
* Update example project to use integer result codes
* check if google pay is available
* pass OPC as a parameter over the bridge

Fixes https://linear.app/alza/issue/ALZ-1938/remove-localmaven

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -59,14 +59,15 @@ android {
...
   }
-
 }
 
 allprojects {
   repositories {
     google()
     mavenCentral()
-    mavenLocal()
+    maven {
```
Comment (david-alza) [inline android/build.gradle:68]: have you confirmed that adding this to alza-mobile will work? might be nice to do that before merging.

File: tapandpay/com/google/android/gms/play-services-tapandpay/18.3.2/play-services-tapandpay-18.3.2.aar.md5 [hunk 2]
```diff
@@ -0,0 +1 @@
+Уџ��2_m��ɻ
```
Comment (david-alza) [inline tapandpay/com/google/android/gms/play-services-tapandpay/18.3.2/play-services-tapandpay-18.3.2.aar.md5:1]: @felipecsl we might not should embed this here. It's a private SDK and we probably don't want this on our public repo.

File: src/index.tsx [hunk 3]
```diff
@@ -17,10 +17,10 @@ const AlzaPaymentPass = NativeModules.AlzaPaymentPass
       }
     );
 
-export function canAddPaymentPass(): Promise<string> {
+export function canAddPaymentPass(): Promise<number> {
```
Comment (david-alza) [inline src/index.tsx:20]: I feel like returning the raw numeric code here makes things quite difficult. Think we could just translate them to a string value?

Comment (felipecsl) [reply src/index.tsx:20]: i don't feel strongly either way. can go back to strings for sure!

File: android/build.gradle (hunk 1 above)
Comment (felipecsl) [reply android/build.gradle:68]: Yeah i've just confirmed with the following steps
1. `npm pack` in this repo
2. Import it to alza-mobile by adding to `package.json`: `"react-native-alza-payment-pass": "file:../alza-react-native-payment-pass/react-native-alza-payment-pass-0.1.1.tgz",`
3. Unzip tapandpay lib to `android/tapandpay`
4. Add `maven { url("$rootDir/tapandpay") }` to build.gradle

**Review by david-alza — APPROVED** [review]

**Pull request merged**